app/models/guide_republisher.rb

```diff
@@ -12,7 +12,7 @@ def republish
 
       publishing_api.put_content(guide.content_id, guide_for_publication.content_payload)
```

Comment:
Thanks for explaining @36degrees - PR is open to fix this here https://github.com/alphagov/service-manual-publisher/pull/399